admin/proc-disabling-telemetry-in-CI-environment.adoc

```diff
@@ -0,0 +1,15 @@
+[id='disabling-telemetry-in-CI-environment_{context}']
```

Comment:
We can remove this section from RHDH docs. THis makes sense only for upstream (showcase)